### PR TITLE
Option to disable the user validation of getters/setters creation

### PR DESCRIPTION
**Pull request by Simounet**
Just put this on your `~/.vimrc` file:
```
let g:vim_php_refactoring_auto_validate = 1
```

### CODE REVIEW

File: plugin/php-refactoring-toolbox.vim [hunk 1]
```diff
@@ -92,8 +96,10 @@ function! PhpCreateSettersAndGetters() " {{{
     for l:property in l:properties
         let l:camelCaseName = substitute(l:property, '^_\?\(.\)', '\U\1', '')
         call s:PhpEchoError('Create set' . l:camelCaseName . '() and get' . l:camelCaseName . '()')
```
Comment (adoy) [inline plugin/php-refactoring-toolbox.vim]: I think this message should only be displayed when the auto_validate is false

**Comment by Simounet** [comment]
The idea was to display the message only if g:vim_php_refactoring_auto_validate is not activated, like your default action.

**Comment by adoy** [comment]
The question is only asked as expected when the option is false, but the message asking for the confirmations is always displayed. It should be in the if

**Comment by Simounet** [comment]
I'm sorry but I really don't get the problem here. I just tested the 2 options and it works as expected. I just get the question only if the boolean is false.

**Pull request merged**